### PR TITLE
Add a new ci task to test recover, reconnect and restart scenario

### PR DESCRIPTION
**Pull request by JeffreyDallas**
**Related issue(s)**:

Regression issue

https://github.com/swirlds/swirlds-platform-regression/issues/2712

### CODE REVIEW

**Review by qnswirlds — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -1459,6 +1459,27 @@ workflows:
               - install-tools
               - attach_workspace:
                   at: /
+  continuous-integration-gcp-restart-misc:
```
Comment (Neeharika-Sompalli) [inline .circleci/config.yml:1462]: Will this run on the some label ?

**Comment by JeffreyDallas** [comment]
@Neeharika-Sompalli so how to trigger this new ci task when adding a new CI label?

File: .circleci/config.yml (hunk 1 above)
Comment (JeffreyDallas) [reply .circleci/config.yml:1462]: How to associate it with a github label ?

Comment (Neeharika-Sompalli) [reply .circleci/config.yml:1462]: As discussed we can run these tests on PR commits.

**Pull request closed without merging**